src/ethereum/utils/block.py

```diff
@@ -0,0 +1,47 @@
+"""
+Utility Functions For Blocks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+..contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Blocks specific utility functions used in this application.
+"""
+from typing import Any, Dict, Tuple
+
+from ethereum.frontier.eth_types import Block, Header
```

Comment:
Hm, I think the same suggestion applies here too. We can't really have `ethereum.frontier` types in `ethereum.utils`. Would an `ethereum.frontier.json` package make sense?